crates/bevy_app/src/main_schedule.rs

```diff
@@ -237,6 +237,30 @@ impl Main {
 pub struct MainSchedulePlugin;
 
 impl Plugin for MainSchedulePlugin {
+    fn init(&self, app: &mut App) {
+        // simple "facilitator" schedules benefit from simpler single threaded scheduling
```

Comment:
I'm not sure whether or not I agree with this, but this change definitely shouldn't be part of this PR.